### ISSUE TITLE
CloudFormation Stack fails due to OpenSSL-version problems

### ISSUE THREAD

**Issue by aholen** [aws-solutions/streaming-data-solution-for-amazon-kinesis-and-amazon-msk#107]
**Describe the bug**
When deploying "Option 4" in [Streaming Data Solution for Amazon Kinesis](https://docs.aws.amazon.com/solutions/latest/streaming-data-solution-for-amazon-kinesis/option4.html) the Stack is stuck on Custom::CreateUUID. Further investigation reveals that the function fails with this error and CloudFormation is stuck on CREATE_IN_PROGRESS:
```
[ERROR] Runtime.ImportModuleError: Unable to import module 'lambda_function': urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'OpenSSL 1.0.2k-fips 26 Jan 2017'. See: https://github.com/urllib3/urllib3/issues/2168
Traceback (most recent call last):

```
**To Reproduce**
Launch the stack as described [here](https://docs.aws.amazon.com/solutions/latest/streaming-data-solution-for-amazon-kinesis/deploy-template4.html)

**Expected behavior**
A stack deployed as described in the page.

**Please complete the following information about the solution:**
- [ ] Version: 1.7.7

To get the version of the solution, you can look at the description of the created CloudFormation stack. For example, "_(SO0124) - AWS Streaming Data Solution for Amazon Kinesis. Version **v1.0.0**_".

- [x] Region: eu-north-1
- [ ] Was the solution modified from the version published on this repository?
- [ ] If the answer to the previous question was yes, are the changes available on GitHub?
- [x] Have you checked your [service quotas](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html) for the sevices this solution uses?
- [x] Were there any errors in the CloudWatch Logs?

**Comment by fhoueto-amz** [comment]
Hi Andreas
Thanks for reporting this. We are analyzing this and will update you as we have more information.
Fabien

**Comment by fhoueto-amz** [comment]
Hi Andreas
This is resolved in 1.7.8.  I am closing the issue. Please feel free to re-open if you have any additional feedback.
Fabien
